run.sh

```diff
@@ -405,15 +387,15 @@ function create_credentials_file(){
 function add_scheduler(){
     echo -e "[INFO] [$(date +"%Y-%m-%d %H:%M:%S")] Add Job Scheduler for run Cloud Function ..."
     
-    job_name="${function_name_prefix}_${function_name_hash}_job"
+    job_name="${function_name}_${function_name_hash}_job"
 
     is_job_scheduler="$(gcloud scheduler jobs list  --location=$gcp_region --filter=$job_name)"
     if [ ! -z "$is_job_scheduler" ]
     then
        gcloud scheduler jobs delete $job_name --location="$gcp_region"
     fi
 
-    function_name_sufix="${function_name_prefix}_${function_name_hash}_func_logzio"
+    function_name_sufix="${function_name}_${function_name_hash}_func_logzio"
```

Comment:
We dont need this as the function name will be a unique through the agent ui

Reply:
Take in a work